connector_prestashop_catalog_manager/models/product_product/common.py

```diff
@@ -5,6 +5,14 @@
 from openerp import models, fields
 
 
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    image_ids = fields.One2many(
+        inverse=False,
```

Comment:
A combination image is exported several times when it's modified:

![jobs](https://user-images.githubusercontent.com/39995504/46291143-04b81180-c58e-11e8-8cf1-002918cb75d0.png)

This is a weird behavior of One2Many inverse and write listener, which is fired once per image field.

The modification of images has been disabled from the combinations to avoid it.